src/sentry/static/sentry/app/views/performance/traceDetails/styles.tsx

```diff
@@ -16,6 +17,13 @@ import {TraceFullDetailed} from 'app/utils/performance/quickTrace/types';
 import {appendTagCondition} from 'app/utils/queryString';
 import {transactionSummaryRouteWithQuery} from 'app/views/performance/transactionSummary/utils';
 
+export {
+  DividerSpacer,
+  ScrollBarContainer as ScrollbarContainer,
+  VirtualScrollBar,
+  VirtualScrollBarGrip,
+} from 'app/components/events/interfaces/spans/header';
```

Comment:
Should the scrollbar manager and its components be moved into module lower in the directory tree? Perhaps `app/components/scrollbarManager`?

Reply:
Yes, the intent is to come back and refactor many of these components out into its own module as trace view shares many of its UI components with the span view. Though I'd like to do that in a separate PR.